src/pages/inquiry/list/index.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { getContactsUs, setContactUsId } from "../../../store/contact-store/contactUsAction";
+import { IContactUs } from "../../../interfaces";
+import BootstrapTable from "react-bootstrap-table-next";
+import ToolkitProvider, { Search } from "react-bootstrap-table2-toolkit";
+import paginationFactory from "react-bootstrap-table2-paginator";
+//import { useHistory } from "react-router-dom";
+
+const InquiryList: React.FC = () => {
+	const dispatch = useDispatch();
+	//const history = useHistory();
+	//const HtmlToReactParser = require("html-to-react").Parser;
+	const state = useSelector((state) => state.contactUsReducer);
+	const contactsUs: IContactUs[] = state.contactsUs;
+	const [selectedTypeContactUs, setselectedTypeContactUs] = useState<IContactUs[]>(contactsUs);
+	const [selectedTab, setSelectedTab] = useState<string>("All");
+
+	// Table confuguration
+	const { SearchBar } = Search;
+	const options = {
+		paginationSize: 4,
+		pageStartIndex: 1,
+		sizePerPage: 15,
+		hideSizePerPage: true,
+		alwaysShowAllBtns: true,
+	};
+
+	// Fetch events information
+	useEffect(() => {
+		dispatch(getContactsUs());
+	}, []);
```

Comment:
This function should write like this. We need to pass the **Skipping props to the array**.
```
useEffect(() => {
   dispatch(getContactsUs());
}, [dispatch, getContactsUs]);
```
The this `useEffect` only run if you call the `getContactsUs()` function. Otherwise it will execute every time when the component rendered which slow down the application performance.